presto-cli/src/main/java/com/facebook/presto/cli/AlignedTablePrinter.java

```diff
@@ -58,6 +58,13 @@ public void finish()
     {
         printRows(ImmutableList.<List<?>>of(), true);
         writer.append(format("(%s row%s)%n", rowCount, (rowCount != 1) ? "s" : ""));
+        flush();
+    }
+
+    @Override
+    public void flush()
```

Comment:
This method actually needs to do the `printRows()` call just like `finish()` does, since that causes it to print the final buffered rows.